### DISCUSSION TITLE
Add different content for tabs with different combinations of dimensions?

### DISCUSSION THREAD

**Discussion by momo91377** [web-infra-dev/rspress#1245] [Ideas]
May rspress consider supporting the implementation of tab switching single-page content with the effect of switching in the link [reference](https://reactnative.dev/docs/set-up-your-environment?os=macos&platform=android), this document(React Native) is based on the Docusaurus, feels like it could improve the quality of documentation for multiple version dependencies.

**Comment by Timeless0911** [comment]
You can write a component, and use css like `hidden` to implement such feature.

Furthermore, PR is welcome to make this component built-in in Rspress.

**Reply by momo91377** [reply]
OK, got it ,thanks
